site/en/blog/first-party-sets-sameparty/index.md

```diff
@@ -0,0 +1,374 @@
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/JNtmMZl7o44LwUk7mE5G.png", alt="ALT_TEXT_HERE", width="800", height="316" %}
+
+First-Party Sets can allow related domain names that are owned and operated by
+the same entity to be treated as first-party in situations where first party and
+third party are otherwise treated differently. The domain names within a
+first-party set are considered _same-party_ and they can label which cookies are
+intended to be set or sent in the same-party context. The aim is to find a
+balance between preventing cross-site tracking by third-parties while still
+maintaining a path that doesn't break valid use-cases.  
+
+The First-Party Sets proposal is currently [in the testing
+phase](https://privacysandbox.com/timeline/), read on to find out how it works
+and how you can try it out.
+
+## What is the difference between first-party and third-party cookies? 
+
+Cookies are not inherently first-party or third-party, it depends on the current
+context where the cookie is included. That's either on a request in the
+``cookie`` header or via ``document.cookie`` in JavaScript.  
+
+If for example, `video.site` has a `theme=dark` cookie, when you're browsing
+`video.site` and a request is made to `video.site`, that's a [same-site
+context](https://web.dev/same-site-same-origin/#same-site-cross-site) and the
+included cookie is _first-party_.  
+
+However, if you're on ``my-blog.site`` which embeds an iframe player for
+``video.site``, when the request is made from `my-blog.site` to `video.site`
+that's cross-site context and the ``theme`` cookie is _third-party_.
+
+{% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/zHa3CTG1l3lYA6C3P433.png", alt="ALT_TEXT_HERE", width="800", height="514" %}
+
+Cookie inclusion is determined by the cookie's ``SameSite`` attribute:
+
++   [**Same-site
+    context**](https://web.dev/same-site-same-origin/#same-site-cross-site) with
+    ``SameSite=Lax``, ``Strict``, or ``None``→ **first-party**
++   **Cross-site context** with ``SameSite=None`` → **third-party**
+
+However, this isn't always so clear cut. Imagine ``brandx.site`` is a travel
+booking site and they also use ``fly-brandx.site`` and ``drive-brandx.site`` to
+separate flights and car hire. Over the course of booking one journey, visitors
+go between these sites to select their different options—they expect their
+"shopping cart" of selections to persist across these sites. ``brandx.site``
+manages the user's session with a ``SameSite=None`` cookie to allow it in
+cross-site contexts. The downside though is now the cookie has no Cross Site
+Request Forgery (CSRF) protection. If ``evil.site`` includes a request to
+``brandx.site`` then it would include that cookie!  
+
+The cookie is cross-site, but all those sites are owned and operated by the same
+organisation. Visitors also understand it's the same organisation and want the
+same session, in other words—a shared identity, across them. With First-Party
```

Comment:
Maybe "With First-Party Sets…" could be some kind of call-out here? Defining this relationship is the core message… it's pretty much the tweetable pull quote, "First-Party Sets provides a route to define situations where a cross-site context is still first-party."